spring-cloud-sleuth-instrumentation/src/main/java/org/springframework/cloud/sleuth/instrument/web/SleuthWebSpan.java

```diff
@@ -78,4 +78,74 @@ public String getKey() {
 
 	}
 
+	/**
+	 * Tags related to security.
+	 *
+	 * @author Marcin Grzejszczak
+	 * @since 3.1.0
+	 */
+	enum SecurityTags implements TagKey {
+
+		/**
+		 * Authorities assigned to the user.
+		 */
+		AUTHORITIES {
+			@Override
+			public String getKey() {
+				return "security.authentication.authorities";
```

Comment:
Would not `security.authorities` be better?
Does it make sense to talk about authorities in terms of authentication should not it belong to authorization?